includes/Admin/Site_Health.php

```diff
@@ -0,0 +1,244 @@
+<?php
+/**
+ * Site Health Class.
+ *
+ * Adds tests and debugging information for Site Health.
+ *
+ * @package   Google\Web_Stories
+ * @copyright 2021 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://github.com/google/web-stories-wp
+ */
+
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+namespace Google\Web_Stories\Admin;
+
+use Google\Web_Stories\Service_Base;
+use Google\Web_Stories\Experiments;
+use Google\Web_Stories\Infrastructure\Conditional;
+
+/**
+ * Class Site_Health
+ *
+ * @package Google\Web_Stories\Admin
+ */
+class Site_Health extends Service_Base implements Conditional {
+	/**
+	 * Experiments instance.
+	 *
+	 * @var Experiments Experiments instance.
+	 */
+	private $experiments;
+
+	/**
+	 * Site_Health constructor.
+	 *
+	 * @since 1.8.0
+	 *
+	 * @param Experiments $experiments Experiments instance.
+	 *
+	 * @return void
+	 */
+	public function __construct( Experiments $experiments ) {
+		$this->experiments = $experiments;
+	}
+
+	/**
+	 * Check whether the conditional object is currently needed.
+	 *
+	 * @since 1.8.0
+	 *
+	 * @return bool Whether the conditional object is needed.
+	 */
+	public static function is_needed() {
+		return is_admin() && ! wp_doing_ajax();
+	}
+
+	/**
+	 * Get the action to use for registering the service.
+	 *
+	 * @since 1.8.0
+	 *
+	 * @return string Registration action to use.
+	 */
+	public static function get_registration_action() {
+		return 'wp_loaded';
+	}
+
+	/**
+	 * Adds the filters.
+	 *
+	 * @since 1.8.0
+	 *
+	 * @return void
+	 */
+	public function register() {
+		add_filter( 'debug_information', [ $this, 'add_debug_information' ] );
+		add_filter( 'site_status_test_php_modules', [ $this, 'add_extensions' ] );
+		add_filter( 'site_status_test_result', [ $this, 'modify_test_result' ] );
+	}
+
+	/**
+	 * Adds debug information for Web stories.
+	 *
+	 * @since 1.8.0
+	 *
+	 * @param array $debugging_information The debugging information from Core.
+	 *
+	 * @return array The debugging information, with added information for Web stories.
+	 */
+	public function add_debug_information( array $debugging_information ) {
+		$extra_data = [
+			'web_stories'             => [
+				'label'       => esc_html__( 'Web Stories Constants', 'web-stories' ),
```

Comment:
```suggestion
				'label'       => esc_html__( 'Web Stories', 'web-stories' ),
```

Reply:
Fixed in https://github.com/google/web-stories-wp/pull/7570/commits/0b0e2cad6be248e8df3e18c5faaebc723f4ae53d